src/code42cli/cmds/profile.py

```diff
@@ -120,28 +149,75 @@ def create(
     debug,
     totp,
 ):
-    """Create profile settings. The first profile created will be the default."""
+    """
+    Create a profile with user token authentication.
+    The first profile created will be the default.
+    """
     cliprofile.create_profile(
-        name, server, username, disable_ssl_errors, use_v2_file_events
+        name,
+        server,
+        username,
+        disable_ssl_errors,
+        use_v2_file_events,
+        api_client_auth=False,
     )
-    password = password or _prompt_for_password(name)
+    password = password or _prompt_for_password()
     if password:
-        _set_pw(name, password, debug, totp=totp)
+        _set_pw(name, password, debug, totp=totp, api_client=False)
+    echo(f"Successfully created profile '{name}'.")
+
+
+@profile.command()
+@name_option(required=True)
+@server_option(required=True)
+@api_client_id_option(required=True)
+@secret_option(required=True)
+@yes_option(hidden=True)
+@disable_ssl_option
+@use_v2_file_events_option
+@debug_option
+def create_api_client(
```

Comment:
I chose to make this a separate `create` command for a couple reasons
- py42's sdk uses two different auth methods
- the `totp` option isn't applicable with api-client auth
- I didn't think an optional password/secret (and in turn prompting the user for authentication on every command) made sense in the case of api clients.

Whether or not this should be a separate command or if we should just add options to the original `create` command is definitely open for discussion.

Reply:
yeah, a separate command makes sense, given there's enough differences. Making it explicit makes it easier to document and harder for a user to do something wrong unintentionally.